it/helpers/WiremockHelper.scala

```diff
@@ -94,15 +94,19 @@ trait WiremockHelper {
   lazy val wmConfig = wireMockConfig().port(wiremockPort)
   lazy val wireMockServer = new WireMockServer(wmConfig)
 
-  def startWiremock() = {
+  def startWiremock(): Unit = {
     wireMockServer.start()
     WireMock.configureFor(wiremockHost, wiremockPort)
   }
 
-  def stopWiremock() = wireMockServer.stop()
+  def stopWiremock(): Unit = wireMockServer.stop()
 
-  def resetWiremock() = WireMock.reset()
+  def resetWiremock(): Unit = WireMock.reset()
 
-  def buildClient(path: String) = ws.url(s"http://localhost:$port/income-tax-view-change$path").withFollowRedirects(false)
+  def buildClient(path: String): WSRequest =
+    ws
+      .url(s"http://localhost:$port/income-tax-view-change$path")
+      .withHttpHeaders("Authorization" -> "Bearer123")
```

Comment:
why do we need authorisation added? I would assume this used to work in the past?

Reply:
It's what I was mentioned Yesterday on the Stand-up call. And I've added the explanation to the Jira ticket (https://jira.tools.tax.service.gov.uk/browse/MISUV-4251): 
If you are migrating to Scala 2.13.8 and see your Integration tests are failing with 401 then, please read the link of how to resolve it
https://hmrcdigital.slack.com/archives/C0J8BH46N/p1655305814832349